bin/git-ignore

```diff
@@ -34,22 +46,22 @@ function add_global {
 }
 
 function show_local {
-  cd "$(git root)"
+  maybe_cd_to_git_root
   show_contents Local .gitignore
 }
 
 function add_local {
-  cd "$(git root)"
+  maybe_cd_to_git_root
   add_patterns .gitignore "$@"
 }
 
 function show_private {
-  cd "$(git root)"
+  maybe_cd_to_git_root
   show_contents Private "${GIT_DIR}/info/exclude"
```

Comment:
We must require a git repo for setting the private ignorefile. As it's private to the git repo.